README.md

```diff
@@ -1,5 +1,9 @@
 # pyoptical
 
+Jon added this!
```

Comment:
can you remove this section @jon, since you already added this to the readme in another pr

Reply:
done